src/refmt_impl.ml

```diff
@@ -92,6 +109,13 @@ let () =
     "-print-width", Arg.Int (fun x -> print_width := Some x), "<print-width>, wrapping width for printing the AST";
     "-heuristics-file", Arg.String (fun x -> heuristics_file := Some x),
     "<path>, load path as a heuristics file to specify which constructors are defined with multi-arguments. Mostly used in removing [@implicit_arity] introduced from OCaml conversion.\n\t\texample.txt:\n\t\tConstructor1\n\t\tConstructor2";
+    "-ocaml-version", Arg.Int (function
```

Comment:
When this file gets brought up to speed with master this can be represented as a Cmdliner enum.